docs/cloud/insights/events-feed.md

```diff
@@ -0,0 +1,59 @@
+<!--
+title: "Events feed"
+sidebar_label: "Events feed"
+custom_edit_url: "https://github.com/netdata/netdata/blob/master/docs/cloud/insights/events-feed.md"
+sidebar_position: "2800"
+learn_status: "Published"
+learn_topic_type: "Concepts"
+learn_rel_path: "Concepts"
+learn_docs_purpose: "Present the Netdata Events feed."
+-->
+
+Netdata Cloud provides the Events feed which is a powerful feature that tracks events that happen on your infrastructure, or in your Space. The feed lets you investigate events that occurred in the past, which is obviously invaluable for troubleshooting. Common use cases are ones like when a node goes offline, and you want to understand what events happened before that. A detailed event history can also assist in attributing sudden pattern changes in a time series to specific changes in your environment.
+
+#### What are the available events?
+
+At a high-level view, these are the domains from which the Events feed will provide visibility into.
+
+> ⚠️ Based on your space's plan different allowances are defined to query past data.
+
+| **Domains of events** | **Community** | **Pro** | **Business** |
+| :-- | :-- | :-- | :-- |
+| **Auditing events** - COMING SOON<br/>Events related to actions done on your Space, e.g. invite user, change user role or change plan.| 4 hours | 7 days | 90 days |
+| **[Topology events](#topology-events)**<br/>Node state transition events, e.g. live or offline.| 4 hours | 7 days | 14 days |
+| **[Alert events](#alert-events)**<br/>Alert state transition events, can be seen as an alert history log.| 4 hours | 7 days | 90 days |
+
+##### Topology events
+
+| **Event name** | **Description** |  **Example** |
+| :-- | :-- | :-- |
```

Comment:
General about the descriptions and examples below. Events are about the past. So all text should be in some form of past tense.

Reply:
@ralphm when writing this, or on one of the discussions around the mapping text, I also thought about that but you could also think of this as a recording of what was happening at that timestamp, so at that timestamp the Node is live/stale/etc.

we need to agree and find a closure on what we want to have on the mappings, changing text here on docs is not an issue since we don't depend on Eng. but on the Netdata Cloud UI is different

we can define what we want and then update [the ticket](https://github.com/netdata/product/issues/1967#user-content-translations) once we close. so alerts I think we are covered what we had on specs for nodes was

> * Nodes state transitions
>    * Unseen / Live / Stale / Offline - Node with <node-name> is on <node-state> state.
>    * Soft Deleted / Hard Deleted / Restored - Node with <node-name> was <node-action>.

what's being suggested:

> * Nodes state transitions
>    * live / stale / offline - Node with <node-name> became <node-state>.
>    * created / deleted (soft) / deleted (hard) / restored - Node with <node-name> was <node-action>.

Reply:
Your examples with angle brackets don't render properly, as markdown sees that as tags.

I'd start all node events with:

> Node {node name} was

So without explicitly saying `with` or `with name`.

I see you continue to mix events and state. ECS actually also has an [`event.kind`](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-kind.html) of `state`, but that's not what we do here. We have `event` and those consistently are about something that happened, so past tense.

As for changes to the texts on the frontend, I'd be happy to put in a PR on those, to lessen the burden on our engineers.

Reply:
> Your examples with angle brackets don't render properly, as markdown sees that as tags.

You're right, changed to the `[ ]` so these can be rendered properly

> I'd start all node events with:

>>Node {node name} was

> So without explicitly saying with or with name.

Will change.

> I see you continue to mix events and state. ECS actually also has an [event.kind](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-kind.html) of state, but that's not what we do here. We have event and those consistently are about something that happened, so past tense.

On this part "I see you continue to mix events and state." I think I hadn't hard about this before, or if it was mentioned on some ticket discussion I may have missed it. With the link you shared it is clear.

On all discussions so far when I mentioned "state" is what we have for our Nodes: Live, Stale, Offline, nothing related to ECS.

> As for changes to the texts on the frontend, I'd be happy to put in a PR on those, to lessen the burden on our engineers.

If you can that would be great.

Reply:
solving this then